src/atoms/connectWalletButton/connectWalletButton.styles.ts

```diff
@@ -0,0 +1,36 @@
+import { Styled } from '../../theme';
+import { ConnectButtonProps } from './types';
+
+export const ConnectButton = Styled.button<ConnectButtonProps>`
+  svg {
+    cursor: pointer;
+    fill: ${(props) => props.theme.textShades.SHADE_MINUS_3}!important;
+  };
+  font-size: 14px;
```

Comment:
Keep in mind to use the Text component for text styling 👍

Reply:
@ArinCantCode I just replicated the same way we were doing things in the regular Button component, but I will look at replacing with a text component